### PR TITLE
fix(tools): Update createStepFile function in new curriculum helper script

### PR DESCRIPTION
**Pull request by RandellDawson**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

This PR updates two helper script utility functions to use the new challenge file format.

### CODE REVIEW

**Comment by ojeytonwilliams** [comment]
@RandellDawson since this adds a bit to the parser, let me know when you're happy with it and I'll clean up the commits.

File: tools/challenge-helper-scripts/utils.js [hunk 1]
```diff
@@ -2,42 +2,86 @@ const fs = require('fs');
...
+${seedHeads}`
+    : '';
+
+  const seedTailSection = seedTails
+    ? `
+
+## --after-user-code--
+
+${seedTails}`
+    : '';
```
Comment (RandellDawson) [inline tools/challenge-helper-scripts/utils.js:81]: @ojeytonwilliams I don't think these lines are needed.  If before/after code is needed, the author of the step can add those later.

Comment (ojeytonwilliams) [reply tools/challenge-helper-scripts/utils.js:81]: True, but my rationale was that those really are part of the seed (it often won't work without them), so it seemed better to include them by default.  Is that not quite right?

Comment (RandellDawson) [reply tools/challenge-helper-scripts/utils.js:81]: Each step is like our current challenges.  We only add those sections when they are needed for the tests of a particular challenge.  The same would go for the before/after sections of a step.

**Review by ojeytonwilliams — APPROVED** [review]

**Review by ahmaxed — APPROVED** [review]

File: tools/challenge-helper-scripts/utils.js (hunk 1 above)
Comment (ojeytonwilliams) [reply tools/challenge-helper-scripts/utils.js:81]: Randy and I discussed this.  There's an argument for including these or not, but for now we're going to include them and see how it works in practice.

**Pull request merged**